### ISSUE TITLE
6.12.0 does not obey some noqa comment tags

### ISSUE THREAD

**Issue by ianw** [ansible/ansible-lint#2970]
With ``test.yaml``` of 

```yaml
- name: Play
  hosts: all
  tasks:
    # We tag this with the unix timestamp so that the upload always
    # has an increasing version number.
    - name: Import sandbox project  # noqa command-instead-of-module
      shell: |
        git clone https://opendev.org/opendev/sandbox
```

```
$ .tox/linters/bin/ansible-lint ./test.yaml 
WARNING: PATH altered to include /home/iwienand/programs/zuul/zuul-jobs/.tox/linters/bin :: This is usually a sign of broken local setup, which can cause unexpected behaviors.
WARNING  Ignore loading rule from /home/iwienand/programs/zuul/zuul-jobs/.tox/linters/lib/python3.11/site-packages/ansiblelint/rules/galaxy.py due to No module named 'pytest'
INFO     Set ANSIBLE_LIBRARY=/home/iwienand/.cache/ansible-compat/7f15ad/modules:/home/iwienand/.ansible/plugins/modules:/usr/share/ansible/plugins/modules
INFO     Set ANSIBLE_COLLECTIONS_PATH=/home/iwienand/.cache/ansible-compat/7f15ad/collections:/home/iwienand/.ansible/collections:/usr/share/ansible/collections
INFO     Set ANSIBLE_ROLES_PATH=/home/iwienand/.cache/ansible-compat/7f15ad/roles:roles:/home/iwienand/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
INFO     Executing syntax check on test.yaml (0.44s)
WARNING  Listing 1 violation(s) that are fatal
test.yaml:6: command-instead-of-module: git used in place of git module
You can skip specific rules or tags by adding them to your configuration file:
# .config/ansible-lint.yml
warn_list:  # or 'skip_list' to silence them completely
  - command-instead-of-module  # Using command rather than module.

                    Rule Violation Summary                    
 count tag                       profile rule associated tags 
     1 command-instead-of-module basic   command-shell, idiom 

Failed after min profile: 1 failure(s), 0 warning(s) on 1 files.
```

This works with 6.11

I bisected this and it started happening @ 497bae917df7739eae885b219440e4be0ee31bcb

**Comment by ianw** [comment]
Another example is (so I guess this shows its more generic than just ```deprecated-command-syntax```)

```yaml
- hosts: all
  pre_tasks:
    # Run ensure-output-dirs now as it is not performed speculatively
    - name: Run ensure-output-dirs
      import_role:
        name: ensure-output-dirs

    - name: Create fake translation  # noqa deprecated-command-syntax
      shell: |
        mkdir -p src/{{ zuul.project.canonical_name }}/translation-source/
        echo Hello > src/{{ zuul.project.canonical_name }}/translation-source/test
```

**Comment by ssbarnea** [comment]
I am afraid that this is unrelated, you missed to add the colons after noqa, `noqa: xxx` works, but `noqa xxx` does not.

I will not close the bug because I spotted two examples in docs that do not use the colon, but all the other ones do.

We need to fix at least the docs. If we are support the syntax without the colon,... not sure, I will ask around.

BTW, Thanks for the good bug reports, keep reporting them.

**Comment by ssbarnea** [comment]
I am working on a hotfix.
